rspace/src/main/scala/coop/rchain/rspace/FreudianSpace.scala

```diff
@@ -0,0 +1,131 @@
+package coop.rchain.rspace
+
+import cats.Id
+import cats.implicits._
+import coop.rchain.catscontrib._
+import coop.rchain.rspace.history.Branch
+import coop.rchain.rspace.internal._
+import coop.rchain.rspace.trace.Log
+import coop.rchain.shared.SyncVarOps
+
+import scala.annotation.tailrec
+import scala.collection.immutable.Seq
+import scala.concurrent.SyncVar
+
+/** The interface for RSpace
+  * It's called a Freudian space because it has a fixed Id
```

Comment:
Letting this pun go through the review phase might in the future be considered a crime against humanity. Just sayin'.